testutil/common/tester.go

```diff
@@ -850,6 +850,12 @@ func (ts *Tester) QueryRewardsBlockReward() (*rewardstypes.QueryBlockRewardRespo
 	return ts.Keepers.Rewards.BlockReward(ts.GoCtx, msg)
 }
 
+// QueryRewardsProviderReward implements 'q rewards block-reward'
+func (ts *Tester) QueryRewardsProviderReward() (*rewardstypes.QueryProviderRewardResponse, error) {
```

Comment:
you're supposed to get the query arguments here: chain ID and provider, and put it in the request object you create a line below ("msg")